src/Simple/JSON.purs

```diff
@@ -143,6 +166,12 @@ instance writeForeignArray :: WriteForeign a => WriteForeign (Array a) where
 instance writeForeignNullOrUndefined :: WriteForeign a => WriteForeign (NullOrUndefined a) where
   writeImpl (NullOrUndefined a) = maybe undefined writeImpl a
 
+instance writeForeignMaybe :: WriteForeign a => WriteForeign (Maybe a) where
+  writeImpl (Just a) = writeImpl a
+  writeImpl Nothing = null
+
+foreign import null :: Foreign
```

Comment:
Could we use the null from nullable? I'd like to not have to deal with FFI in this package...

Reply:
I thought about that and decided to go down this route in lieu of adding another dependency. That said, I'm fine with either.

Reply:
nullable is an indirect dependency anyway, better use that one.

Reply:
Ah, `null` isn't exposed by Nullable but really, I'd even prefer `toForeign $ toNullable Nothing`